README.md

```diff
@@ -35,11 +35,11 @@ To get a local copy up and running follow these simple example steps.
 - Open `index.html` in the `build` directory.
 - Enjoy!
 
-### Usage
+<!-- ### Usage
 
 ### Run tests
 
-### Deployment
+### Deployment -->
```

Comment:
- [ ] Kindly remove commented code. If you do not have the necessary information now, you can add it at another time.